app/src/main/res/layout/lessons_completed_chapter_view.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <import type="org.oppia.android.app.model.ChapterPlayState" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.topic.lessons.ChapterSummaryViewModel" />
+  </data>
+
+  <LinearLayout
+    android:id="@+id/lessons_completed_chapter_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:onClick="@{() -> viewModel.onClick(viewModel.explorationId)}"
+    android:orientation="vertical">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/color_def_dark_green"
+      android:minHeight="48dp"
+      android:orientation="horizontal">
+
+      <TextView
+        android:id="@+id/chapter_index"
+        android:layout_width="60dp"
```

Comment:
We should refrain from giving fixed widths to `TextView's` .

Suggest making this `wrap_content` and using `minWidth=60dp` instead.

Reply:
done